server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const app = express()
+const port = 3000
```

Comment:
1. Хорошим тоном считается называть целочисленные константы в UPPERCASE. Так при чтении кода сразу будет понятно, что это значение не должно меняться
2. Очень полезно бывает указывать порт через переменные окружения (ты столкнешься с этим, когда будешь разворачивать приложение в докере)
```suggestion
const PORT = process.env.PORT || 3000
```